pkg/component/kubernetes/apiserverexposure/sni.go

```diff
@@ -48,6 +50,8 @@ const (
 	// authenticationDynamicMetadataKeyAPIServerProxy is the key used to configure the istio envoy filter for the APIServer proxy.
 	authenticationDynamicMetadataKeyAPIServerProxy = "authenticated-shoot"
 
+	connectionUpgradeRouteName = "connection-upgrade"
```

Comment:
Nit: It is not strictly required, but some of the surrounding constants also have a documentation string. Maybe we want that here as well because it is not obvious for me from the constant name.
WDYT?